README.md

```diff
@@ -84,6 +84,17 @@ development. Test Kitchen also exposes the ability to provision instances using
 various cloud providers like AWS, DigitalOcean, or OpenStack. For more
 information, please see the [Test Kitchen documentation](http://kitchen.ci).
 
+For Windows instances, you need to take two extra steps:
+
+1. Modify .kitchen.yml so that the "chef" project has the correct chef_omnibus_root:
+   ```
+   - name: chef
+     provisioner:
+       chef_omnibus_install_options: -P angrychef
+       chef_omnibus_root: /opscode/angrychef
```

Comment:
C:/opscode/angrychef

using just /opscode may select the default drive on which this command is being run which may be fine but it's better to be explicit.